bazel/envoy_library.bzl

```diff
@@ -29,6 +29,25 @@ def envoy_basic_cc_library(name, deps = [], external_deps = [], **kargs):
         **kargs
     )
 
+# All Envoy extensions must be tagged with their security hardening stance with
+# respect to downstream and upstream data plane threats. These are verbose
+# labels intended to make clear the trust that operators may place in
+# extensions.
+EXTENSION_SECURITY_POSTURES = [
+    "robust_to_untrusted_downstream",
+    "robust_to_untrusted_upstream",
+    "robust_to_untrusted_upstream_and_upstream",
```

Comment:
`upstream_and_upstream`

Reply:
Beyond this typo, can you add a comment to each one to make it super pedantically clear what this means? Or link to somewhere else that explains it?

Reply:
It should presumably be `robust_to_untrusted_upstream_and_downstream`. But the existence of this typo might also point to the fact that this name becomes a bit of tangle to read.

Do you want to add a posture specifically relevant to the WASM and Lua filters?

Reply:
Thanks for the typo clarification. I rather this be super explicit, so will just fix the wording. If we use generic terms like "trusted" or "untrusted", there is a danger the reader becomes confused IMHO.